distro/adaptation/opensuse

```diff
@@ -50,7 +50,7 @@ libaio-dev: libaio-devel
 libapache2-mpm-itk: apache2
 libapparmor-dev: libapparmor-devel
 libarchive-dev: libarchive-devel
-libarchive-tools: libarchive13 bsdtar
+libarchive-tools: libarchive bsdtar
```

Comment:
There is a limitation here... Because this change is done in the value of the key-value pair, it will be taken as is. Hence even if the correct mapping of `libarchive: libarchive13` is added to this file, it will not be replaced.

Reply:
Since the installation script does not recursively search for any other packages in the values to be replaced, this change will break `lkp install` on `opensuse/leap`. (OpenSUSE doesn't have `libarchive`, but has `libarchive13` instead)

Reply:
> Since the installation script does not recursively search for any other packages in the values to be replaced, this change will break `lkp install` on `opensuse/leap`. (OpenSUSE doesn't have `libarchive`, but has `libarchive13` instead)

For this, shall we undo this change or should we raise an issue?

Reply:
You can undo it and force push a new commit.

Reply:
> You can undo it and force push a new commit.

Alright, thanks for the clarification 👍